pages/common/qmv.md

```diff
@@ -0,0 +1,24 @@
+# qmv
+
+> Move files and directories with your favorite text editor.
+> More information: <https://www.nongnu.org/renameutils/>.
+
+- Move a single file (invoke an editor with source on the left, target on the right):
+
+`qmv {{source_file}}`
+
+- Move multiple JPG files:
+
+`qmv {{*.jpg}}`
+
+- Move directories:
+
+`qmv --directory {{my_directory/}} {{another_directory/}} {{the_other_directory/}}`
+
+- Move files/directories inside a directory:
+
+`qmv --recursive {{my_directory/}}`
```

Comment:
How about `qmv -R {{directory}}`?

Reply:
Same as above, I already use short option in 166264880efc5de6535bc741e6df7f4f10e5020f.

By the way, do I need to remove trailing slash after directory name?

Reply:
I have tried the command, it seems with or without trailing slash, the result is same. So let's remove it? Or let more people to review this command.

Reply:
Trailing slash is just my personal style (since I observe for a long time that almost all command treat directory with or without argument the same way). I put it there to make it obvious that the argument is a directory.